components/Members/MembersCompactWrapper.tsx

```diff
@@ -8,13 +8,22 @@ import MemberOverview from './MemberOverview'
 import { PlusIcon } from '@heroicons/react/outline'
 import AddMember from './AddMember'
 import useGovernanceAssets from '@hooks/useGovernanceAssets'
+import Tooltip from '@components/Tooltip'
 
 const MembersCompactWrapper = () => {
-  const { symbol, councilMint } = useRealm()
+  const {
+    symbol,
+    councilMint,
+    toManyCouncilOutstandingProposalsForUse,
```

Comment:
nit: `tooManyCouncilOutstandingProposalsForUse`. and `tooManyCommunityOutstandingProposalsForUser`